features/checkout/having_registration_account_form_prefilled_after_checkout.feature

```diff
@@ -0,0 +1,31 @@
+@checkout
+Feature: Having registration form prefilled after checkout
+    In order to make future purchases with ease
+    As an Visitor
+    I want to have account registration form prefilled after placing an order
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store has a product "PHP T-Shirt" priced at "$19.99"
+        And the store ships everywhere for free
+        And the store allows paying offline
+
+    @ui
+    Scenario: Having prefilled registration form after checkout
+        Given I have product "PHP T-Shirt" in the cart
+        And I complete addressing step with email "john@example.com" and "United States" based shipping address
+        And I proceed with "Free" shipping method and "Offline" payment
+        And I confirm my order
+        Then I should see the thank you page
+        And I should be able to proceed to the registration
+        And this button should redirect to registration form prefilled with "john@example.com" email
```

Comment:
`this button` referenced to unclear place on the website. IMHO scenario is good without this line